app/boards/shields/sofle_rgb_keyhive/Kconfig.defconfig

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2022 Josh Whitley
```

Comment:
Generally, we prefer to use `The ZMK Contributors` as the copyright holder for the repository to reduce churn on future edits. This is an optional change of course, no problem if you'd prefer to keep it as is.

Reply:
Updated all copyrights to be generic.